src/api/search/README.md

```diff
@@ -0,0 +1,37 @@
+# Search Service
+
+The Search Service provides a direct querying controller which interfaces with our ELK stack.
+
+## Install
+
+```
+npm install
+```
+
+## Usage
+
+```
+# normal mode
```

Comment:
Do you want to add a section about [running in dev mode](https://github.com/Seneca-CDOT/telescope/blob/master/src/api/users/README.md#usage)?

Reply:
I don't have anything on this cos I can't even run it myself yet...